sql/testdata/explain_plan

```diff
@@ -34,6 +34,20 @@ Level  Type    Description
 query ITT colnames
 EXPLAIN SELECT * FROM t LIMIT 1 OFFSET 1
 ----
-Level  Type  Description
-0      limit count: 1, offset:  1
-1      scan  t@primary
+Level  Type         Description
+0      presentation limit: 1, offset:  1
+1      scan         t@primary
+
+query ITT colnames
+EXPLAIN SELECT DISTINCT * FROM t
+----
+Level  Type         Description
+0      presentation distinct
+1      scan         t@primary
+
+query ITT colnames
+EXPLAIN SELECT DISTINCT * FROM t LIMIT 1 OFFSET 1
```

Comment:
We need a test that executes (and not just EXPLAINs) a combination of DISTINCT and LIMIT/OFFSET, to make sure that DISTINCT is applied first.